src/screens/transaction-address.tsx

```diff
@@ -24,12 +27,12 @@ export const TransactionAddressScreen = () => {
     }
 
     if (!address) {
-      return wallets.snapshot();
+      return wallets;
     }
 
-    return wallets
-      .filtered('address CONTAINS[c] $0 or name CONTAINS[c] $0', address)
-      .snapshot();
+    return wallets.filter(
+      w => w.address.includes(address) || w.name.includes(address),
```

Comment:
Please add the same behavior like operator [c] in Realm, which stands for 'case insensitivity'

Reply:
Fixed